evap/results/templates/result_bar.html

```diff
@@ -6,17 +6,15 @@
         {% if participants_warning %}
             class="participants-warning"
         {% endif %}
-        {% if result.total_count %}
-            data-toggle="tooltip" data-placement="left" title="{% if participants_warning %}{% trans 'Only a few participants answered this question.' %}</br></br>{% endif %}
-            {% for count in result.counts %}
-                {% with answer_name=result.question|get_answer_name:forloop.counter %}
-                    {% if answer_name %}
-                        {{ answer_name }}: {{ count }}/{{ result.total_count }} ({{ count|percentage_one_decimal:result.total_count }})
-                        {% if not forloop.last %}</br>{% endif %}
-                    {% endif %}
-                {% endwith %}
-            {% endfor %}"
-        {% endif %}
+        data-toggle="tooltip" data-placement="left" title="{% if participants_warning %}{% trans 'Only a few participants answered this question.' %}</br></br>{% endif %}
```

Comment:
yeah, it's still possible in this commit, although not anymore in one of the later commits. @janno42 needs to tell me what this check was supposed to do.

Reply:
i think this was introduced to prevent the case @He3lixxx described. if total_count is 0, no tooltip or warning "only a few..." should be displayed at any point. i don't know why we didn't check for if result.counts, though.
anyway - it's probably easier to just remove this check and find out during the review that something should be changed than trying to figure out what exactly led to this specific line of code. looks like we should have added a comment there ;)

Reply:
this is not about questions having no answers, this case is handled by the surrounding `if result.average is not None`. we already talked about this and you said this might be for single results, but didn't remember in which case they would have zero answers.

Reply:
upon reviewing this again, i don't see how this check can have any effect. before the changes in this PR, if total_count was 0, then average and counts are both None. this should be handled by the surrounding if. after this PR, total_count, counts and average are either all three None or all three larger than zero. @He3lixxx could you confirm?

Reply:
As I see it, yes, that is correct.